src/foam/u2/detail/AbstractSectionedDetailView.js

```diff
@@ -102,6 +106,214 @@ foam.CLASS({
 
         return sections;
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'loading',
+      documentation: `True while we're calculating 'sections'.`,
+      expression: function(outstandingCalculations_) {
+        return outstandingCalculations_ > 0;
+      }
+    },
+    {
+      class: 'FObjectArray',
+      of: 'foam.layout.Section',
+      name: 'sections',
+      factory: function() {
+        return this.sections_;
+      }
+    },
+    {
+      class: 'Long',
+      name: 'outstandingCalculations_',
+      documentation: 'Used alongside `loading` to hide the view while it is loading',
+      factory: function() {
```

Comment:
Why is this factory needed?

Reply:
I don't think it is. Was just sanity checking while debugging and forgot to remove this afterwards.

Reply:
Addressed by 1d8e5c759.